app/lib/clients/vault/entity.rb

```diff
@@ -0,0 +1,17 @@
+module Clients
+  class Vault
+    class << self
+      def put_entity(name, policies)
+        client.logical.write("identity/entity",
+                             name: name,
+                             policies: policies)
+      end
+      def read_entity(name)
```

Comment:
i'm surprised rubocop doesn't complain here, but I think it looks better to have a blank line between function defs.